pulp_service/pulp_service/app/constants.py

```diff
@@ -7,6 +7,10 @@
     rpm="Red Hat", npm="npm", maven="Maven", python="PyPI", gem="RubyGems"
 )
 
+# Labels used in RPM repositories for vulnerability report tasks
+OSV_RH_ECOSYSTEM_LABEL = "osv_dev ecosystem"
+OSV_RH_ECOSYSTEM_CPES_LABEL = "osv_dev cpes"
```

Comment:
```suggestion
OSV_RH_ECOSYSTEM_CPES_LABEL = "osv.dev cpes"
```

Is using a period an option?

Reply:
Unfortunately, not: https://github.com/pulp/pulpcore/blob/9b6c47ffa88377de814c47c2ceb0cf819cf6f288/pulpcore/app/serializers/fields.py#L430

I also had some issues with pulp-cli: https://github.com/pulp/pulp-cli/issues/1165